cam0:
  T_cam_imu:
  - [0.999975, 0.007038, -0.000812, -0.037983]
  - [-0.007035, 0.999969, 0.003395, 0.002513]
  - [0.000836, -0.003389, 0.999994, -0.001260]
  - [0.000000, 0.000000, 0.000000, 1.000000]
  cam_overlap: [1]
  camera_model: pinhole
  distortion_coeffs: [0.008701, -0.018283, 0.038756, -0.025935]
  distortion_model: equidistant
  intrinsics: [469.650280, 469.551357, 375.629402, 236.808264]
  resolution: [752, 480]
  rostopic: /cam0/image_raw
depth:
  T_depth_cam0:
  - [0.999967, 0.007459, -0.003335, 0.059147]
  - [-0.007466, 0.999970, -0.002064, -0.032501]
  - [0.003319, 0.002089, 0.999992, -0.007220]
  - [0.000000, 0.000000, 0.000000, 1.000000]
  distortion_coeffs: [0.019387, -2.214324, -0.001496, -0.001959, 3.901117]
  distortion_model: radtan
  intrinsics: [215.255792, 215.334327, 109.314966, 89.491694]
  depth_correction: [0.071772, -0.077959, 0.005362, 0.019133, 0.002183, 0.006643]
  resolution: [224,171]
vicon:
  T_vicon_cam:
  - [0.999444, 0.028517, -0.017273, 0.049830]
  - [-0.028686, 0.999542, -0.009611, -0.004717]
  - [0.016991, 0.010101, 0.999805, 0.004368]
  - [0.000000, 0.000000, 0.000000, 1.000000]
